index.src.html

```diff
@@ -1733,11 +1733,12 @@
 <xmp class='idl'>
 dictionary VideoEncoderConfig {
   required DOMString codec;
-  [EnforceRange] unsigned long long bitrate;
   [EnforceRange] required unsigned long width;
   [EnforceRange] required unsigned long height;
   [EnforceRange] unsigned long displayWidth;
   [EnforceRange] unsigned long displayHeight;
+  [EnforceRange] unsigned long long bitrate;
+  [EnforceRange] double framerate;
```

Comment:
Usual framerates are not representable with doubles. I don't think this matters here in theory (since it's for bitrate tuning), but it's always annoying.

Reply:
Yeah, I hear that. Assumed it wasn't worth the trouble to do a rational here. Sort of similar to MediaCapabilities framerate (just used for performance estimation).

Reply:
We used double for `maxFramerate` in [WebRTC-Extensions](https://w3c.github.io/webrtc-extensions/#dom-rtcrtpencodingparameters-maxframerate).